types/Config.js

```diff
@@ -53,10 +53,41 @@ export type DefaultConfig = {|
   watch: boolean,
 |};
 
-export type Config = DefaultConfig & {|
```

Comment:
This was changed for a reason. Intersection doesn't work well with exact types: https://github.com/facebook/flow/issues/2626